src/app/series/directives/series-podcast.component.html

```diff
@@ -117,21 +117,20 @@
       <input type="text" readonly [ngModel]="podcast?.publishedUrl" name="publishedUrl"/>
     </prx-fancy-field>
 
-    <prx-fancy-field label="Public Feed Url" class="feed-url">
-      <div *ngIf="podcast?.hasPublicFeed">
-        <div class="fancy-hint">
-          The public URL for your podcast feed. Feel free to share this URL with listeners. If you need to alter this URL, please
-          do so safely by using the New Feed Url field in the Advanced settings.
-        </div>
-        <input type="text" readonly [ngModel]="podcast?.publicFeedUrl" name="publicFeedUrl" #pubFeed/>
-        <button [publishCopyInput]="pubFeed">Copy</button>
-        <a class="button" target="_blank" rel="noopener" [href]="podcast?.publicFeedUrl">Open Link</a>
```

Comment:
Advanced Confirm can also be user indepedently. From styleguide, for reference: `<input
              prxAdvancedConfirm="That's a risky move" 
              [prxModel]="model" 
              prxName="foo" 
              (ngModelChange)="model.set('foo', $event)"
              [ngModel]="model.foo">`